### DISCUSSION TITLE
How can I read the Test Outcome or the Message object from the test results. I want to read the object from the dispose method

### DISCUSSION THREAD

**Discussion by RonyDaherS1** [xunit/xunit#2178] [Question] [answered]
If we are using Assert.true(x = y, errorMessage) in the xunit test. I would like to capture the assert message (in case assert failed) and the test outcome from the dispose method.

![TestResult](https://user-images.githubusercontent.com/5772349/97449002-9af2c900-1907-11eb-88af-53fe06516185.JPG)

**Comment by bradwilson** [comment, marked as answer]
There's no way to do this today in the 2.x code base, but I suspect that test context per #621 will be the way to do this eventually in 3.0.

**Reply by bradwilson** [reply]
Going to mark this as answered. Please let me know if you have any further questions.
